plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -317,6 +317,10 @@ class WC_Install {
 	 * Hook in tabs.
 	 */
 	public static function init() {
+		if ( ! empty( $GLOBALS['wc_uninstalling_plugin'] ) ) {
```

Comment:
I think it's not overly likely that another plugin will be using the installer class when they're uninstalling, but if we want to be extra cautious without introducing a new global, we can still use `WP_UNINSTALL_PLUGIN`.

`WP_UNINSTALL_PLUGIN` is [defined with the plugin uninstall file](https://github.com/WordPress/wordpress-develop/blob/1e04fc0864544f5a224dbe4718979367ab2a208c/src/wp-admin/includes/plugin.php#L1311) and we could check to make sure it matches the current plugin directory.

Reply:
Just adding that I'm leaving this as-is per our discussion.